src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+---
+label: Maintaining consistent and engaging digital interfaces throughout IBM, whether applications or experiences, demands extended guidance around color usage.
+title: Color
+tabs: ['Overview', 'Usage']
+---
+
+## Get started
+
+<grid-wrapper col_lg="8" flex="true" bleed="true">
+<clickable-tile
+    title="Color Palettes for Adobe™(.ase)"
+    href="https://w3.ibm.com/design/language//IBM-color-palettes.zip"
+    type="resource"
+    >
+    <img src="images/ase.png" alt="Color palettes for Adobe" />
+</clickable-tile>
+</grid-wrapper>
+
+<anchor-links>
+<ul>
+    <li><a href="#introduction">Introduction</a></li>
+    <li><a href="#color-anatomy">Color anatomy</a></li>
+    <li><a href="#ui-color-themes">UI color themes</a></li>
+    <li><a href="#interaction-states">Interaction states</a></li>
+    <li><a href="#digital-accessibility">Digital accessibility</a></li>
+</ul>
+</anchor-links>
+
+## Introduction
+
+Application of our color palette brings an unified and recognizable consistency to IBM’s vast array of digital products and interfaces. This consistency is grounded by a set of well-defined rules on how to work with our component library in the context of dark and light themes.
+
+
+![Ui screens](images/IBM_UI_Screens.gif)
+
+
+## Color anatomy
+
+IBM Product Design uses a subset of the IBM Design Language color palette. The Neutral Gray family is dominant in our UI, making use of subtle shifts in value to help organize content into distinct zones. 
+
+The core Blue family has been designated as the primary action color across all products and experiences, ensuring our color aesthetic is a part of every interaction. Additional colors are used sparingly and purposefully.
+
+![Color anatomy](images/color_anatomy_bkg.svg)
+
+### Color terms
+
+| Term      | Definition                                                                                                  |
+| --------- | ----------------------------------------------------------------------------------------------------------- |
+| Theme | The set of unique values assigned to each token.                                                            |
+| Token | The code identifier for a unique role or set of roles. Tokens are universal and never change across themes. |
+| Role | The systematic usage/s of a value. Roles cannot be changed between themes.                                  |
+| Value | The actual style (ie hex code) assigned to a token.                         |
+
+
+## UI Color themes
+
+Our palette is designed and implemented in a themable manner. Each theme works with a specific set of primary background colors. These colors are global variables which determine the color of the component being applied within each theme. The light theme uses White and Gray 10 backgrounds and the dark theme uses Gray 100, 90 and 80 backgrounds. Default color tokens are provided for each component based on the primary background color.
```

Comment:
This whole section is a little iffy and not quite how the themes are actually going to work.

Potential rewrite:
"Our palette is designed and implemented in a themable manner. Each theme works with a specific primary background color. This main global variable influences the color of the component being applied within each theme. There are four themes, two light and two dark. The two light themes use White and Gray 10 backgrounds and the dark themes use Gray 90 and Gray 100 backgrounds."

Reply:
I think this first sentence could use some work too. The first person isn't really appropriate in docs unless there is a compelling reason to use it. . . And "in a themable manner" is awkward as well. I'd propose "The Carbon palette can be easily customized by using _themes_." Also, overall this paragraph is quite choppy, with a bunch of short sentences back to back. I'm happy to take a stab at reworking the paragraph.

Reply:
I think the [Language](https://w3.ibm.com/design/language/elements/color/) site writes a lot in the first person @claycrenshaw might want to pass that feedback up to them so we're all taking in the same manner

Reply:
Well, first person is more appropriate when talking about a design language, as it's coming from the POV of a corporate brand. It's a subtle difference for sure. But Carbon isn't really speaking for "IBM" the way the design language is.

Reply:
On second thought, maybe we should make an exception for this page. The entire page is definitely written from an "IBM Design Language" point of view, so maybe we just go with it... Unlike a lot of the other pages on the site, this one is really about IBM-specific stuff at its core.